src/Test/Spec.purs

```diff
@@ -37,7 +37,6 @@ instance eqResult :: Eq Result where
   eq Success Success = true
   eq (Failure _) (Failure _) = true
   eq _ _ = false
-  eq r1 r2 = not (r1 == r2)
```

Comment:
This is a remnant from the update for 0.7. This was previously the (/=) definition which I forgot to remove.